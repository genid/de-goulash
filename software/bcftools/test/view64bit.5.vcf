##fileformat=VCFv4.2
##INFO=<ID=END,Number=A,Type=Integer,Description="dummy">
##INFO=<ID=MPOS,Number=A,Type=Integer,Description="dummy">
##FORMAT=<ID=XX,Number=A,Type=Integer,Description="dummy">
##contig=<ID=chr1,length=248956422>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	S1
chr1	1	.	G	C	.	.	END=42949672,1,5000000000;MPOS=100000000	XX	1,-2147483640,-2147483641,-2147483647,-2147483648,2147483647,2147483648
chr1	1	.	G	C	.	.	END=-2147483640	XX	-2147483640
chr1	1	.	G	C	.	.	END=-2147483641	XX	-2147483641
chr1	1	.	G	C	.	.	END=-2147483647	XX	-2147483647
chr1	1	.	G	C	.	.	END=-2147483648	XX	-2147483648
chr1	1	.	G	C	.	.	END=-2147483649	XX	-2147483649
