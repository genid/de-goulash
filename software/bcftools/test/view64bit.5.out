chr1	1	.	G	C	.	.	END=42949672,1,.;MPOS=100000000	XX	1,-2147483640,.,.,.,2147483647,.
chr1	1	.	G	C	.	.	END=-2147483640	XX	-2147483640
chr1	1	.	G	C	.	.	END=.	XX	.
chr1	1	.	G	C	.	.	END=.	XX	.
chr1	1	.	G	C	.	.	END=.	XX	.
chr1	1	.	G	C	.	.	END=.	XX	.
